resources/js/Components/NewBullet.vue

```diff
@@ -24,20 +24,8 @@
                 placeholder="Unburden your mind..."
                 @keydown.up="up"
                 @keydown.down="down"
-                @keydown.enter="
-                    if ($event.shiftKey) {
-                        return
-                    }
-                    $event.preventDefault()
-                    if ($event.target.value.trim() !== '') {
-                        create()
-                    }
-                "
-                @blur="
-                    if ($event.target.value.trim() !== '') {
-                        create()
-                    }
-                "
+                @keydown.enter.prevent="create"
```

Comment:
```suggestion
                @keydown.enter.exact.prevent="create"
```

Does this still fix this issue for you? This change would allow Shift+Enter to add a newline.

Reply:
I wasn't able to verify the fix as it only happens on mobile devices. It was kind of a stab in the dark but it made sense (at least in my head). There are ways to remote debug using a mobile device but it's a bit of a hassle setting it all up. I thought the Shift + Enter in inputs was a native thing?

Reply:
Wasn't aware of the .exat modifier, but that seems a good alternative!

Reply:
Wasn't aware of the .exact modifier, but that seems a good alternative!

Reply:
You can give this a merge and I'll test it out. The claen up (with the .exact modifier) is still good even if it doesn't fix the issue.

Reply:
You can give this a merge and I'll test it out. The clean up (with the .exact modifier) is still good, even if it doesn't fix the issue.